NahuriraClintonie/Sacco-Frontend
# Kimwanyi sacco user login page

**What Does The PR do?**
**PR Title:** Implement Login Page with Registration Link
**PR Description:**
This pull request adds a login page to the project that allows users to input their username and password. Additionally, it includes a link to the registration page for users who don't have an account yet.

**Description of Task to be completed?**
The task for this pull request is to implement a login page with the following functionalities:
    1. Create a login form with input fields for username and password.
    2. Add a "Login" button to submit the form.
    3. Include a link to the registration page for new users.
    
 **How should this be manually tested?**
To manually test the login page, follow these steps:
    1. Open the login page in a web browser.
    2. Enter a valid username and password.
    3. Click the "Login" button and ensure that the user is redirected to the desired destination (e.g., a dashboard page).
    4. Click the registration link and verify that it redirects to the registration page.

**Any background context you want to provide?**
This PR aims to provide a basic login functionality for users and facilitate the registration process for new users who don't have an account yet.

**Screenshots**
![kimwanyi login](https://github.com/NahuriraClintonie/Sacco-Frontend/assets/98154281/e5dc0bb2-c256-46b0-a358-f1b17a381429)
**
